### ISSUE TITLE
Broken text formatting in lists

### ISSUE THREAD

**Issue by RarrugasTetra** [asciidoctor/asciidoctor-bibtex#52]
Since upgrading to asciidoctor 2.0.10 I am getting broken formatting of text in lists if my adoc file is processed with asciidoctor-bibtex.

Minimal example of source file:
```
= Test

* Item1, `code`
* Item2, *bold*
* Item3, _I_~sub~
```
If compiled as 
`asciidoctor -r asciidoctor-bibtex test.adoc`

gives

```
Test

    Item1, <code>code</code>

    Item2, <strong>bold</strong>

    Item3, <em>I</em><sub>sub</sub>

```

If I compile it without bibtex as
`asciidoctor test.adoc`
it gives a correct html with appropriately formatted list items.

My versions:
```
$ gem list
asciidoctor (2.0.10, 1.5.8)
asciidoctor-bibtex (0.5.1)
```

```
$ asciidoctor -v
Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.5.5p157 (2019-03-15 revision 67260) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)

```

Is this a bug?

**Comment by ProgramFan** [comment]
Does this happen with earlier asciidoctor and asciidoctor-bibtex 0.5.1? asciidoctor-bibtex does not change the list item text when no citation is found. So these lines in your example shall not change before feeding to the convertor.

**Comment by RarrugasTetra** [comment]
With asciidoctor 1.5.x everything was (is) fine. The presence or absence of citations does not play role. My real file does contain some of citations and I have the titled problem. The minimal example above has no citations and the problem still exists. But it appears **only** if I compile with asciidoctor-bibtex.

**Comment by ProgramFan** [comment]
I see. This bug is solved in 0.5.2.
